api/rosetta/data_integration_test.go

```diff
@@ -203,33 +238,33 @@ func TestGetBalanceBadRequest(t *testing.T) {
 	assert.Equal(t, http.StatusBadRequest, e.Code)
 }
 
-// getBalanceRequest will generate a BalanceRequest with the specified parameters.
-func getBalanceRequest(address string, blockIndex uint64, blockHash string) rosetta.BalanceRequest {
+// balanceRequest will generate a BalanceRequest with the specified parameters.
```

Comment:
It still bothers me that we use the future tense everywhere when there's no need for it 😅 It will not do anything, it does it right now! [General truth](https://www.grammaring.com/tags/grammar-tags/general-truth#:~:text=The%20present%20simple%20is%20used,boils%20at%20100%20degrees%20Celsius.)! :p 

```suggestion
// balanceRequest generates a BalanceRequest with the specified parameters.
```